example/csv/vector_polygon.csv

```diff
@@ -0,0 +1,3 @@
+prefix,location,epsg
+feature_geojson,example/geojson/WaiteOvalTest1.geojson,4326
```

Comment:
I don't think you need the epsg column here.

Reply:
All vector based data sources should have the CRS embedded in the file so Hoang is technically correct but I am inclined to ask for this in this file explicitly as a means of checking if the data is provided in the same reference system as the provider intends. I think its best to leave in but lets discuss.